README.md

```diff
@@ -4,96 +4,18 @@ Welcome to the DevCycle .NET Server SDK, which interfaces with a local bucketing
 Events are queued and flushed periodically in the background.
 This version uses .NET Standard 2.1 and utilizes more resources to perform local bucketing.
 
-## Requirements
-
-
-### Frameworks supported
-- .NET & .NET Core >=3.0
-- Mono >=6.4
-- Xamarin.iOS >=12.16
-- Xamarin.Mac >=5.16
-- Xamarin.Android >=10.0
-- Unity >=2021.2
-
-### Dependencies
-- FubarCoder.RestSharp.Portable.Core >=4.0.8
-- FubarCoder.RestSharp.Portable.HttpClient >=4.0.8
-- JsonSubTypes >=1.8.0
-- Microsoft.Extensions.Logging.Abstractions >= 6.0.1
-- Microsoft.Extensions.Logging.Console >= 6.0.1
-- Microsoft.Extensions.Logging >= 6.0.0
-- Newtonsoft.Json >=13.0.1
-- TypeSupport >= 1.1.12
-- Wasmtime >= 0.34.0-preview1
-
 ## Installation
-Download the SDK from Nuget - https://www.nuget.org/packages/DevCycle.DotNet.Server.Local.SDK/1.0.2
+Download the SDK from Nuget - https://www.nuget.org/packages/DevCycle.DotNet.Server.Local.SDK/1.7.0
 and use the namespaces:
 ```csharp
```

Comment:
Wrong namespaces here

Reply:
eliminated the namespace list from this section